<launch>
	<param name="num_blocks" value="4" type="int"/>
	<param name="configuration" value="stacked_descendingadsfasdfa" />
	<param name="sim" value="false" />

	<node name="robot_interface" pkg="zic_proj1" type="robot_interface.py" output="screen">
	</node>
	<node name="controller" pkg="zic_proj1" type="controller.py" />
</launch>
